webpack.config.js

```diff
@@ -3,6 +3,13 @@ let productionBuild = (process.env.production_build == 1);
 let version = "v0.0.1";
 let plugins = [];
 
+plugins.push(
+    new webpack.DefinePlugin({
+        'process.env.NODE_ENV': productionBuild ? JSON.stringify('production') : JSON.stringify('development'),
+        'global': {}
+    })
+);
+
```

Comment:
Yes! For all the dependcies that `imports` or `requires`, it will just bundle them all into the output js file